src/region.js

```diff
@@ -84,9 +84,9 @@ const Region = MarionetteObject.extend({
     }
   },
 
-  _attachView(view) {
+  _attachView(view, {replaceElement} = {}) {
     const shouldTriggerAttach = !view._isAttached && isNodeAttached(this.el);
-    const shouldReplaceEl = !!this.getOption('replaceElement');
+    const shouldReplaceEl = !!replaceElement || !!this.getOption('replaceElement');
```

Comment:
Ahh yeah, you're right. That's what I get for trying to copy the existing "!!" style without thinking.